supreme/verifier/src/commonMain/kotlin/AttestationVerifier.kt

```diff
@@ -86,7 +86,7 @@ class AttestationVerifier(
             androidAttestationConfiguration.attestationStatementValiditySeconds
         ),
         nonceGenerator: NonceGenerator = suspend { CryptoRand.nextBytes(ByteArray(64)) },
-        challengeValidator: ChallengeValidator = InMemoryChallengeCache(clock, verificationTimeOffset)
+        challengeValidator: ChallengeValidator = InMemoryChallengeCache(clock, -verificationTimeOffset)
```

Comment:
What if `InMemryChallengeCache` always takes the absolute value of this parameter and then subtracts it? So the sign wouldn't matter.